source/if-any.d.ts

```diff
@@ -0,0 +1,29 @@
+import type {IsAny} from './is-any';
+
+/**
+An `If`/`Else`-like type that resolves depending on whether the given type is `any`.
+
+If the given type `T` is `any`, the returned type is `TypeIfAny`. Otherwise, the return type is `TypeIfNotAny`. If only `T` is specified, `TypeIfAny` will be `true` and `TypeIfNotAny` will be false.
+
+@see IsAny
```

Comment:
Can you check if this is linkified in VS Code? We need a syntax that like to the type correctly so users can just click through.

Reply:
How about only using `@link`? https://devblogs.microsoft.com/typescript/announcing-typescript-4-3/#jsdoc-link-tags

Reply:
I think having `@see` makes it clearer.

With:

<p align="center"><img width="488" alt="image" src="https://user-images.githubusercontent.com/1403701/229916617-53c4e33b-42fb-44e4-b111-945fd130f0fe.png"></p>

Without:

<p align="center"><img width="487" alt="image" src="https://user-images.githubusercontent.com/1403701/229916679-44eef508-652e-44f1-8938-310efa450d1b.png"></p>